slackbot-guysgals-reasoning.md

```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
```

Comment:
Ah, I see what you mean and I agree. I'll switch it around, more along the lines of "Certain words can be alienating to various members of the ZATech community, in particular...."

As discussed elsewhere a small paragraph on "even if you have no intention of being alienating a lot of people who aren't men find it othering etc etc"